site/index.html

```diff
@@ -5,8 +5,9 @@
 
   <meta charset="utf-8">
   <!--[if IE]><meta http-equiv='X-UA-Compatible' content='IE=edge,chrome=1'><![endif]-->
-  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
-  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="description" content="Theia is an open-source cloud &nbsp; desktop IDE framework implemented in TypeScript.">

```

Comment:
Note: This is the description of https://github.com/theia-ide/theia + "open-source".